MVCapplication/src/main/java/com/softserve/if072/mvcapp/controller/ProductPageController.java

```diff
@@ -35,19 +38,14 @@
 
 @Controller
 @RequestMapping("/product")
+@PropertySource(value = {"classpath:application.properties", "classpath:message.properties"})
 public class ProductPageController extends BaseController {
 
-    @Value("${application.restProductURL}")
-    private String productUrl;
+    @Autowired
+    private ProductPageService productPageService;
```

Comment:
Few links for reading
https://www.petrikainulainen.net/software-development/design/why-i-changed-my-mind-about-field-injection/
http://vojtechruzicka.com/field-dependency-injection-considered-harmful/
http://aleksz-programming.blogspot.com/2014/03/why-you-should-autowire-constructor.html

is it clear, or should I explain?